channels/hn.m3u

```diff
@@ -45,10 +57,12 @@ https://emisoras.hn:8081/kerussotv/index.m3u8
 http://lencatelevision.com:8080/hls/ltv1.m3u8
 #EXTINF:-1 tvg-id="LTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/dprA1P7.png" group-title="",LTV (720p) [Not 24/7]
 https://5e85d90130e77.streamlock.net/6022/6022/playlist.m3u8
-#EXTINF:-1 tvg-id="MetroTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",MetroTV (720p)
+#EXTINF:-1 tvg-id="Maya.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/91/A6287BAC-05B5-4739-9415-01B08CC0DF0E.png" group-title="Local",Maya TV
+https://media.streambrothers.com:1936/8140/8140/playlist.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/147909664-172deefb-505a-4bf0-9b4c-f026ad270d0b.png)

Reply:
![my](https://user-images.githubusercontent.com/30985701/147956550-65897b3d-06a2-4816-9544-c711c5a02988.jpg)

Reply:
Not 24/7, couple of hours ago it was offline